src/entities/candidate/candidate.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Candidate } from './candidate.entity';
+
+@Injectable()
+export class CandidateService {
+  constructor(
+    @InjectRepository(Candidate)
```

Comment:
Please, make candidateRepository property readonly in the constructor.

Reply:
Done, and fixed this for properties in another services and controllers